src/NiceHashMiner/translations.json

```diff
@@ -2918,14 +3265,16 @@
       "ro": "Am integrat programe de minare 3rd party care ar trebui să crească viteza de minare și să vă dea o experiență de minare mai stabilă - acest lucru poate duce la mai multe câștiguri pe o perioadă mai scurtă de timp chiar și după ce taxa dezvoltatorului este scăzută. Totuși, deoarece acestea sunt programe 3rd party care nu sunt open-source, nu avem nici o posibilitate de a le inspecta. NiceHash nu poate garanta că folosirea acestor programe și nu își asumă nici o responsabilitate pentru orice daune cauzate, breșe de securitate, pierderi de date și fonduri, erori de sistem sau hardware, și orice alte probleme. Acceptând această declinare de responsabilitate vă asumați întreaga răspundere pentru utilizarea acestori mineri fără open-source așa cum sunt.",
       "ru": "Мы интегрировали стороннее программное обеспечение от Claymore, которое должно поднять скорость и дать вам более стабильный опыт майнинга — это потенциально может привести к большему заработку за короткий промежуток времени даже после вычета комиссии разработчика. Но поскольку это ПО является полностью закрытым, у нас нет никакой возможности исследовать его. NiceHash не может ручаться за использование этого ПО и отказывается брать на себя ответственность за любой причинённый ущерб — нарушение безопасности, потерю данных и средств или другие проблемы. Соглашаясь с этим уведомлением, вы берёте на себя полную ответственность за использование этих майнеров с закрытым исходным кодом от Claymore.",
-      "zh_cn": "我们集成立了第三方挖矿程序。这些程序将会加快部分算法的运算速度并给您更为稳定的挖矿体验。即便考虑到这些程序会索取小额的第三方使用费，它们在短时间内仍将为您带来更高的收益。然而我们需要指出这些程序是闭源的第三方软件，因此我们无法检视它们的源码。NiceHash无法为这些软件提供认证，也无法对任何因使用第三方软件而造成的数据或财产损失，信息安全隐患，软件或硬件故障，以及任何其他损害或问题承担责任。如果您同意此免责条款，您将对由因第三方闭源软件的使用而所致的一切后果承担责任。"
+      "zh_cn": "我们集成立了第三方挖矿程序。这些程序将会加快部分算法的运算速度并给您更为稳定的挖矿体验。即便考虑到这些程序会索取小额的第三方使用费，它们在短时间内仍将为您带来更高的收益。然而我们需要指出这些程序是闭源的第三方软件，因此我们无法检视它们的源码。NiceHash无法为这些软件提供认证，也无法对任何因使用第三方软件而造成的数据或财产损失，信息安全隐患，软件或硬件故障，以及任何其他损害或问题承担责任。如果您同意此免责条款，您将对由因第三方闭源软件的使用而所致的一切后果承担责任。",
+      "de": "Wir haben Drittanbieter Mining-Software integriert, die dein Mining verschnellern und eine stabilere Erfahrung bieten sollte - das kann zu einen höheren Verdienst in einer geringeren Zeit führen, selbst wenn die Entwicklergebühr abgezogen wurde. Da die Drittanbieter Software nicht quelloffen ist, kann diese nicht von uns untersucht werden. NiceHash kann für die Nutzung dieser Software nicht bürgen und übernimmt keinerlei Verantwortung für jegliche verursachte Schäden, Sicherheitsvorfälle, Daten- oder Guthabenverluste, System- oder Hardwarefehler und etwaige andere Probleme. Mit der Zustimmung zu diesem Haftungsausschluss übernimmst du die volle Verantwortung für die Nutzung der nicht quelloffenen Miner so wie sie sind."
```

Comment:
Wir haben Drittanbieter Mining-Software integriert, die dein Mining verschnellern und eine stabilere Erfahrung bieten sollte - das kann zu einen höheren Verdienst in einer geringeren Zeit führen, selbst wenn die Entwicklergebühr abgezogen wurde. Da die Drittanbieter Software nicht OpenSource ist, kann diese nicht von uns untersucht werden. NiceHash kann für die Nutzung dieser Software nicht bürgen und übernimmt keinerlei Verantwortung für jegliche verursachte Schäden, Sicherheitsvorfälle, Daten- oder Guthabenverluste, System- oder Hardwarefehler und etwaige andere Probleme. Mit der Zustimmung zu diesem Haftungsausschluss übernimmst du die volle Verantwortung für die Nutzung der nicht OpenSource Miner so wie sie sind.